src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/sql.js

```diff
@@ -0,0 +1,274 @@
+loadScript('https://cdnjs.cloudflare.com/ajax/libs/sql-formatter/4.0.2/sql-formatter.min.js')
+
+class SqlVisualization extends Visualization {
+    static inputType = 'Any' // 'Standard.Database.Data.Table.Table | Standard.Database.Data.Column.Column' // TODO change this once sum types are properly supported
```

Comment:
1. Too long line, max 100 chars everywhere - apply to ALL files
2. In comments always add links to issues describing them.
3. Comments should be valid english sentences. missing dot

Apply to all your code.